x-pack/plugins/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -0,0 +1,393 @@
+                          id="xpack.ml.explorer.limitLabel"
+                          defaultMessage="Limit"
+                        />
+                      </span>
+                    }
+                    display={'columnCompressed'}
+                  >
+                    <SelectLimit />
+                  </EuiFormRow>
+                </EuiFlexItem>
+                <EuiFlexItem grow={false} style={{ alignSelf: 'center' }}>
+                  <EuiFormRow label="&#8203;">
+                    <div className="panel-sub-title">
+                      {viewByLoadedForTimeFormatted && (
+                        <FormattedMessage
+                          id="xpack.ml.explorer.sortedByMaxAnomalyScoreForTimeFormattedLabel"
+                          defaultMessage="(Sorted by max anomaly score for {viewByLoadedForTimeFormatted})"
+                          values={{ viewByLoadedForTimeFormatted }}
+                        />
+                      )}
+                      {viewByLoadedForTimeFormatted === undefined && (
+                        <FormattedMessage
+                          id="xpack.ml.explorer.sortedByMaxAnomalyScoreLabel"
+                          defaultMessage="(Sorted by max anomaly score)"
+                        />
+                      )}
+                      {filterActive === true && viewBySwimlaneFieldName === VIEW_BY_JOB_LABEL && (
+                        <FormattedMessage
+                          id="xpack.ml.explorer.jobScoreAcrossAllInfluencersLabel"
+                          defaultMessage="(Job score across all influencers)"
+                        />
+                      )}
+                    </div>
+                  </EuiFormRow>
+                </EuiFlexItem>
+              </>
+            )}
+
+            {menuItems.length > 0 && (
+              <EuiFlexItem grow={false} style={{ marginLeft: 'auto' }}>
+                <EuiPopover
+                  button={
+                    <EuiButtonIcon
+                      size="s"
+                      aria-label={i18n.translate('xpack.ml.explorer.swimlaneActions', {
+                        defaultMessage: 'Actions',
+                      })}
+                      color="subdued"
+                      iconType="boxesHorizontal"
+                      onClick={() => {
+                        setIsMenuOpen(!isMenuOpen);
+                      }}
+                    />
+                  }
+                  isOpen={isMenuOpen}
+                  closePopover={() => {
+                    setIsMenuOpen(false);
+                  }}
+                  panelPaddingSize="none"
+                  anchorPosition="downLeft"
+                >
+                  <EuiContextMenuPanel items={menuItems} />
+                </EuiPopover>
+              </EuiFlexItem>
+            )}
+          </EuiFlexGroup>
+
+          <EuiSpacer size="m" />
+
+          <div
+            className="ml-explorer-swimlane euiText"
```

Comment:
Nit: With this refactor it might be an opportunity to rename this class to `mlExplorerSwimlane` across all usages.